<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMailWindow</class>
 <widget class="QMainWindow" name="NewMailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fromLabel">
        <property name="text">
         <string>From: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fromLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="toLabel">
        <property name="text">
         <string>To: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="toLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="subjectLabel">
        <property name="text">
         <string>Subject:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="subjectLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ccLabel">
        <property name="text">
         <string>CC:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ccLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMessage">
    <property name="title">
     <string>&amp;Message</string>
    </property>
    <addaction name="actionSend"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edi&amp;t</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuMessage"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSend"/>
  </widget>
  <widget class="QToolBar" name="toolBarRichText">
   <property name="windowTitle">
    <string>Richt Text Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionUnderlinded"/>
  </widget>
  <action name="actionSend">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/res/icons/mail-send.svg</normaloff>:/icons/res/icons/mail-send.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Send</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/res/icons/format-text-bold.svg</normaloff>:/icons/res/icons/format-text-bold.svg</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUnderlinded">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/res/icons/format-text-underline.svg</normaloff>:/icons/res/icons/format-text-underline.svg</iconset>
   </property>
   <property name="text">
    <string>Underlinded</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
